ask-smapi-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ask-smapi-sdk",
-  "version": "1.1.0",
+  "version": "1.1.1",
```

Comment:
However, it's a feature for internal usage, rather than public facing feature. What do you think ? @breedloj

Reply:
You guys are both right. My gut told me patch version since it is an internal utility and not a customer facing feature. I am OK with either but would lean towards patch unless we have other changes to go out with it.